### PR TITLE
The Jupiter font does not contain some content that would cause it to become"???"

### PR DESCRIPTION
**Pull request by AinaSnow**
The Jupiter font does not contain some content that would cause it to become"???",It is recommended to use the axis of the game itself

d6fb2b49036432ece76f0d1c4e0e69e1
ac8a5414c615743a5c82f73455149ed2
After repair
b999bb598d70772c6d2beb3c27f285c5
4cb83d1a8e2c3c18f0f05d3f2606ef8a

### CODE REVIEW

**Comment by AinaSnow** [comment]
Actually because Jupiter doesn't include some fonts, all default set names under jp become "????"

File: Umbra/src/Toolbar/Widgets/Location/LocationWidget.Elements.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ internal partial class LocationWidget
                         size: new(250, 14),
                         text: "Zone Name Here",
                         style: new() {
-                            Font         = Font.AxisSmall,
```
Comment (haroldiedema) [inline Umbra/src/Toolbar/Widgets/Location/LocationWidget.Elements.cs:40]: Is there a specific reason for this change? If not can you leave this one at `AxisSmall` so it stays aligned with the Weather portion of the widget?

**Review by haroldiedema — CHANGES_REQUESTED** [review]

File: Umbra/src/Toolbar/Widgets/Location/LocationWidget.Elements.cs (hunk 1 above)
Comment (AinaSnow) [reply Umbra/src/Toolbar/Widgets/Location/LocationWidget.Elements.cs:40]: Maybe I made a mistake. You can continue to use AxisSmall. In fact, the only two things that need to be changed are Jupiter➡AxisLarge. I first tried to use Axis to replace the undisplayable Jupiter, but he was too small.

**Review by haroldiedema — APPROVED** [review]

**Comment by haroldiedema** [comment]
Thanks! 👍

**Pull request merged**